buildSrc/bump-version.js

```diff
@@ -76,12 +77,17 @@ async function bumpIosVersion(newVersionString) {
 /**
  * @param currentVersion {number[]}
  * @param newVersionString {string}
+ * @param buildGradlePath {string}
  * @return {Promise<void>}
  */
-async function bumpAndroidVersion(currentVersion, newVersionString) {
-	const buildGradlePath = "app-android/app/build.gradle"
+async function bumpAndroidVersion(currentVersion, newVersionString, buildGradlePath) {
 	const buildGradleString = await fs.promises.readFile(buildGradlePath, "utf8")
-	const oldVersionCodeMatch = buildGradleString.match(/versionCode (\d+)/)
+
+	const kotlinRegex = /versionCode = (\d+)/
```

Comment:
just a thought: we should put it somewhere in gradle.properties or something, where it's easier to parse